p04-mui-quotes/src/Components/Quote/Quotes.js

```diff
@@ -1,41 +1,105 @@
 import { Grid, IconButton, Typography } from '@mui/material'
-import React from 'react'
+import {React, useState, useEffect, useRef} from 'react'
 import CurrentQuote from './CurrentQuote'
+import axios from 'axios';
 
-import { Favorite, SkipNext, FavoriteBorderOutlined } from '@mui/icons-material';
+import { Favorite, SkipNext, FavoriteBorderOutlined, Pause, Timer } from '@mui/icons-material';
 
 import { QuoteActions, QuoteContainer, QuoteListContainer } from './Quotes.styles'
 import QuotesFavsList from './QuotesFavsList';
+import configData from '../../Configs/config.json';
 
 const Quotes = () => {
+  const [randomQuoteTimer, setRandomQuoteTimer] = useState(0)
+  const [content, setContent] = useState('')
+  const [author, setAuthor] = useState('')
+  const [year, setYear] = useState('')
+  const [tags, setTags] = useState([])
+  const [pause, setPause] = useState(false)
+  const [favorite, setFavorite] = useState(false)
+  const [favoriteList, setFavoriteList] = useState([])
 
+  const myTimeout = useRef(null)
+
+  const fetchRandomQuote = () => {
+    const url = configData.QUOTABLE_API_URL + "/random"
+    axios.get(url).then((response) => {
+      setContent(response.data.content)
+      setAuthor(response.data.author);
+      setYear(response.data.dateAdded.substr(0, 4))
```

Comment:
It's perfectly good to substring the date in order to subtract the year, just to let you know that you can use the Date function of javascript 

```
setYear(new Date(response.data.dateAdded).getFullYear())
```